tests/framework/extensions/provisioning/verify.go

```diff
@@ -338,7 +338,7 @@ func VerifyHostnameLength(t *testing.T, client *rancher.Client, clusterObject *s
 		query2, err := url.ParseQuery(fmt.Sprintf("labelSelector=%s=%s", capi.MachineDeploymentNameLabel, md.Name))
 		require.NoError(t, err)
 
-		machineResp, err := client.Steve.SteveType(machineNameAnnotation).List(query2)
+		machineResp, err := client.Steve.SteveType("cluster.x-k8s.io.machine").List(query2)
```

Comment:
can we make this a const?

Reply:
Yes, moved the const from ssh file to verify as that's more generic, and addressed